mozilla_schema_generator/schema.py

```diff
@@ -34,11 +34,14 @@ class Schema(object):
     def __init__(self, schema: dict):
         self.schema = schema
 
-    def set_schema_elem(self, key: Tuple[str], elem: Any) -> dict:
+    def set_schema_elem(self, key: Tuple[str], elem: Any, *, propogate=True) -> dict:
```

Comment:
A  comment on `propagate` would be useful. It looks like it's the equivalent of `mkdir -p` for a nested folder where a parent doesn't exist.